tasks/systemd.yml

```diff
@@ -0,0 +1,47 @@
+- name: Make sure systemd unit file directory does exist
+  file:
+    path: /lib/systemd/system/
+    mode: "u=rwX,g=rX,o=rX"
+    owner: root
+    group: root
+    state: directory
+
+- name: Install systemd service file (vm@.service)
+  template:
+    dest: /lib/systemd/system/vm@.service
+    group: root
+    owner: root
+    mode: 'u=rw,g=r,o=r'
+    src: vm@.service.j2
+
+- name: Install systemd service file (vm_iscsi@.service)
+  template:
+    dest: /lib/systemd/system/vm_iscsi@.service
+    group: root
+    owner: root
+    mode: 'u=rw,g=r,o=r'
+    src: vm_iscsi@.service.j2
+
+- name: Create listing folder for all vms requiring iscsi(needed for selective vm_iscsi@.service starts)
+  file:
+    path: "{{ xen_vman_vm_config_path }}/.iscsi/"
```

Comment:
Is it convention or default to use a hidden folder?
If it's not much trouble to change this, I'd rather have a visible folder.

Reply:
Or - make the directory name a variable